core/src/foundations/ionic.root.scss

```diff
@@ -0,0 +1,118 @@
+/**
+ * This is an auto-generated file, please do not change it directly.
+ * Ionic Design System
+ */
+
+:root {
+--ionic-color-primary-10: #f7faff;
```

Comment:
~ Non-blocking, we should see if we can format this or generate the output to match our indentation for CSS variables.